checkov/terraform/checks/resource/base_resource_negative_value_check.py

```diff
@@ -0,0 +1,60 @@
+from abc import abstractmethod
+
+import dpath
+
+from checkov.common.models.enums import CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class BaseResourceNegativeValueCheck(BaseResourceCheck):
+    def __init__(self, name, id, categories, supported_resources):
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        excluded_key = self.get_excluded_key()
+        if excluded_key is not None:
+            if dpath.search(conf, excluded_key) != {}:
+                value = dpath.get(conf, excluded_key)
+                if isinstance(value, list) and len(value) == 1:
+                    value = value[0]
+                if self.check_excluded_condition(value):
+                    return CheckResult.PASSED
+
+        inspected_key = self.get_inspected_key()
+        bad_values = self.get_forbidden_values()
+        if dpath.search(conf, inspected_key) != {}:
+            value = dpath.get(conf, inspected_key)
+            if isinstance(value, list) and len(value) == 1:
+                value = value[0]
+            if value in bad_values or len(bad_values) == 0:
```

Comment:
@rotemavni Please use Checkov's `ANY_VALUE` instead of the empty list

Reply:
Done